core/src/processing/opengl/PSurfaceJOGL.java

```diff
@@ -195,7 +195,8 @@ protected void initGL() {
     if (profile == null) {
       if (PJOGL.profile == 2) {
         try {
-          profile = GLProfile.getGL2ES1();
+          // request at least GL2 or GLES2
+          profile = GLProfile.getGL2ES2();
```

Comment:
But it sounds like that might take a little while, so let's add a platform check there and get the ES2 profile if it's running on ARM, and we can use that for the time being until #3505 is finished.